lib/build_pack.rb

```diff
@@ -0,0 +1,12 @@
+require 'fileutils'
+require 'net/http'
+
+module BuildPack
+  def self.run(build_dir, cache_dir)
+    Installer.install(build_dir, cache_dir)
+  end
+end
+
+require_relative "build_pack/installer"
+require_relative "build_pack/downloader"
+require_relative "build_pack/logger"
```

Comment:
I usually put all the requires up at the top. Also I'd prefer to include the other libs in the file they are used in rather than here. e.g  require net/http in the downloader file